piet-cairo/src/text.rs

```diff
@@ -71,14 +78,30 @@ impl Text for CairoText {
     }
 
     fn load_font(&mut self, _data: &[u8]) -> Result<FontFamily, Error> {
+        /*
+         * NOTE(ForLoveOfCats): It does not appear that Pango natively supports loading font
+         * data raw. All online resource I've seen so far point to registering fonts with
+         * fontconfig and then letting Pango grab it from there but they all assume you have
+         * a font file path which we do not have here.
+         * See: https://gitlab.freedesktop.org/fontconfig/fontconfig/-/issues/12
+         */
```

Comment:
It might be possible to load a font with freetype then hand off an `FT_Face` to pango. Not sure but possibly

Reply:
Looking some more, it seems pango really wants to push you down the path of using fontconfig. I [found 1 example](https://stackoverflow.com/questions/10722577/loading-a-font-from-a-file-to-pango) of manually creating a pango font description from a FcFace.